Application/global_settings_manager.py

```diff
@@ -86,3 +97,53 @@ def load_global_settings(self):
 
         settings.endGroup()
         settings.endGroup()
+        settings.endGroup()
+
+    def set_to_user_settings(self):
```

Comment:
This method should be renamed, "save_user_settings". Also, to make this manager more readable and general-purpose, the existing method "update_saved_settings" should be renamed to something more specific to its actual functionality (saving button definitions). We should probably rename the method to "save_encoding_button_definitions"